### PR TITLE
feat: Paid drop followups

### LINKED ISSUE #8936: Paid drop followups
last week it was quite a rush, lets finish it, it will make the ux even better 

## Things to add 
we are currently missing the "why?" tooltip
image
image

- tooltip copy is the same for bot warnings 
image

cc @hassnian wanna finish? 👀

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #8936

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-01-11 at 09 26 19

### CODE REVIEW

File: assets/styles/global.scss [hunk 1]
```diff
@@ -396,6 +396,12 @@ a.has-text-grey {
   }
 }
 
+.has-text-k-orange3 {
```
Comment (preschian) [inline assets/styles/global.scss:399]: use tailwind, don't add more classes on global.scss

File: components/drops/useDrops.ts [hunk 2]
```diff
@@ -115,11 +114,12 @@ export const useDropStatus = (id: string) => {
   }
 }
 
+const MINIMUM_FUNDS_ROUND = 4
```
Comment (vikiival) [inline components/drops/useDrops.ts:117]: This should be unified across the app imo cc @exezbcz

**Review by vikiival — COMMENTED** [review]
~

File: components/drops/useDrops.ts (hunk 2 above)
Comment (yangwao) [reply components/drops/useDrops.ts:117]: breaking?

Comment (Jarsen136) [reply components/drops/useDrops.ts:117]: I think not

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

will test soon on canary with drops and we can push it live for today drop, let's see!

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.48 USD/DOT ~ 3.538 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0x53ed822325d361405f23c9d3ce6ff96eeda23d4e84c64d2690bf497c9555c1d6](https://polkadot.subscan.io/extrinsic/0x53ed822325d361405f23c9d3ce6ff96eeda23d4e84c64d2690bf497c9555c1d6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**